### PR TITLE
workflow: retry arxiv download from non-export url INSPIR-237

### PR DESCRIPTION
**Pull request by puntonim**
https://its.cern.ch/jira/browse/INSPIR-237

### CODE REVIEW

File: inspirehep/modules/workflows/tasks/arxiv.py [hunk 1]
```diff
@@ -67,7 +67,17 @@ def populate_arxiv_document(obj, eng):
...
+    if not is_valid_pdf_link:
+        # Try with the alternative arxiv url.
+        alternative_url = current_app.config['ARXIV_PDF_URL_ALTERNATIVE']\
+            .format(arxiv_id=arxiv_id)
+        is_valid_pdf_link = is_pdf_link(alternative_url)
+        if is_valid_pdf_link:
+            url = alternative_url
+
+    if not is_valid_pdf_link:
         if NO_PDF_ON_ARXIV in requests.get(url).content:
```
Comment (szymonlopaciuk) [inline inspirehep/modules/workflows/tasks/arxiv.py]: Maybe it worth to run this check also on the result of the first request? If the author didn't make the source available then it's probably not worth querying the second URL. Also then you could put this code in a for-loop or similar?

Comment (puntonim) [reply inspirehep/modules/workflows/tasks/arxiv.py]: Ok I will try with the for loop.

Comment (puntonim) [reply inspirehep/modules/workflows/tasks/arxiv.py]: Done

**Comment by puntonim** [comment]
Done

**Pull request merged**